spek-api/src/main/kotlin/org/jetbrains/spek/api/IgnoreSpek.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.spek.api
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS)
+annotation class IgnoreSpek
```

Comment:
`Ignore` would be too common and might conflict with other annotations. But probably we could use that since kotlin allows import aliasing.